src/test_workflow/test_recorder/test_recorder.py

```diff
@@ -46,17 +54,45 @@ def _generate_std_files(self, stdout: str, stderr: str, output_path: str) -> Non
             stderr_file.write(stderr)
 
     def _generate_yml(self, test_result_data: TestResultData, output_path: str) -> str:
+        base_file_path = self._get_file_path(self.base_path, test_result_data.component_name, test_result_data.component_test_config)
+
+        components_files = self._get_list_files(output_path)
+        components_files = self._update_absolute_file_paths(components_files, base_file_path, "")
+
+        local_cluster_logs_path = os.path.join(output_path, "local-cluster-logs")
+        local_cluster_logs = self._get_list_files(local_cluster_logs_path)
+        local_cluster_logs = self._update_absolute_file_paths(local_cluster_logs, base_file_path, "local-cluster-logs")
+
+        opensearch_service_logs_path = os.path.join(local_cluster_logs_path, "opensearch-service-logs")
+        opensearch_service_logs = self._get_list_files(opensearch_service_logs_path)
+        opensearch_service_logs = self._update_absolute_file_paths(opensearch_service_logs, base_file_path, os.path.join("local-cluster-logs", "opensearch-service-logs"))
+
+        test_result_file = components_files + [
+            {
+                "local-cluster-logs":
+                    local_cluster_logs + opensearch_service_logs
+            }
+        ]
+
         outcome = {
             "test_type": self.test_type,
-            "test_run_id": self.test_run_id,
+            "run_id": self.test_run_id,
```

Comment:
Why the rename?

Reply:
I want this key to be written above the `test_result_files` when generating the yaml file but not strong opinion to change it. Open to change it back.